tests/PHPUnit/PredisConnectionTestCase.php

```diff
@@ -80,11 +80,12 @@ public function testSupportsSchemeUnix()
 
     /**
      * @group disconnected
-     * @expectedException \InvalidArgumentException
-     * @expectedExceptionMessage Invalid scheme: 'udp'.
      */
     public function testThrowsExceptionOnInvalidScheme()
     {
+        $this->expectException('InvalidArgumentException');
```

Comment:
Shouldn't this use `InvalidArgumentException::class` everywhere, instead of strings?

Reply:
It has the same behavior as a string. The scope of the PR was to fix the test suite for PHPUnit 8.

Reply:
> Shouldn't this use `InvalidArgumentException::class` everywhere, instead of strings?

@mfn fixes related to PHPUnit are the sole focus of this PR so I'd say to consider this for a different PR along with other improvements to the test suite.

That said I wouldn't be against this change and it should be applied also to the plenty of places we use `getMockBuilder()` and `assertInstanceOf()` probably, but is there any tangible advantage (aside from potentially having shorter lines when a class / interface is in the same namespace or has been _imported_) that makes it worth the hassle?